cluster/definition.go

```diff
@@ -155,6 +162,10 @@ func (d Definition) Verify() error {
 		}
 	}
 
+	if noSigs > 0 && noSigs != len(d.Operators) {
```

Comment:
so, either no one signs OR all of them sign, only then we consider it verified?

Reply:
yup